pkg/network/vmispec/interface_test.go

```diff
@@ -126,6 +126,91 @@ var _ = Describe("VMI network spec", func() {
 			Entry("mid", vmiStatusInterfaces(iface1, netName, iface2)),
 		)
 	})
+
+	Context("migratable", func() {
+		const (
+			migratablePlugin    = "mig"
+			nonMigratablePlugin = "non_mig"
+			podNet0             = "default"
+		)
+
+		bindingPlugins := map[string]v1.InterfaceBindingPlugin{
+			migratablePlugin:    {Migration: &v1.InterfaceBindingMigration{}},
+			nonMigratablePlugin: {},
+		}
+
+		Context("binding plugin", func() {
+			It("there is no pod network", func() {
+				const nonPodNet = "nonPodNet"
+				networks := []v1.Network{
+					{Name: nonPodNet, NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{}}},
+				}
+				ifaces := []v1.Interface{interfaceWithBridgeBinding(nonPodNet)}
+				Expect(netvmispec.IsMigratableBindingPlugin(networks, ifaces, bindingPlugins)).To(BeFalse())
```

Comment:
We do not need `IsMigratableBindingPlugin` exposed, we should be able to check this with `VerifyVMIMigratable` which uses it.

Reply:
We are checking it using `VerifyVMIMigratable`. But I find this method useful and wirth being exposed although it is not externally used ATM.  Also, by exposing it I can test it separately.

Reply:
I do not understand why there is a need to expose it if it is not used anywhere else and tests can be examined through another function.